tools/ZAPDConfigs/gc-eu/Config.xml

```diff
@@ -6,4 +6,5 @@
     <SpecialEntranceList File="../SpecialEntranceList.txt"/>
     <EnumData File="../EnumData.xml"/>
     <ExternalXMLFolder Path="assets/xml/"/>
+    <ExternalFile XmlPath="objects/gameplay_keep_pal.xml" OutPath="assets/objects/gameplay_keep/"/>
```

Comment:
I'd rather see this in each xml that needs it, since not all assets reference gkeep, nor does it make sense for all assets to have gkeep available in their relevant memory map

Also new-assets co-extracts assets that reference each other this way in order to use the extra information for extraction. Making all assets co-extract with gkeep would prevent multiprocessing

Reply:
Yeah this is not great, but otherwise I think we'd have to duplicate hundreds of XMLs to either reference `objects/gameplay_keep_pal.xml` or `objects/gameplay_keep.xml` depending on the version...

Reply:
Ah I didn't catch that
Yeah ok let's go like this (gkeep in global config) for now then. :(